shared/utils.py

```diff
@@ -12,16 +11,19 @@
 
 from rsbeams.rsplot import util
 from rsbeams.rsdata.SDDS import readSDDS
+
 # from rsbeams.rsplot import beam_plots
 
-#dict for screens, their sizes, and resolution
-screens_dict = OrderedDict({'TCPhosphor':[.01,.000037],"ChicaneSlit":[.01,.00003378],"DCPhosphor":[0.01,.0000168],
-                'Phosphor1':[.01,.0000165],"Aline1":[.01,.00002217],"Aline2":[0.01,.00002394],
-                'Aline3':[.01,.0000244],"VisaEBeam1":[.0015,.00000755],"VisaEBeam2":[0.0015,.00000755],
-                'VisaEBeam3':[.0015,.00000752],"VisaEBeam4":[.0015,.00000741],"VisaEBeam5":[0.0015,.00000768],
-                'VisaEBeam6':[.0015,.00000763],"VisaEBeam7":[.0015,.00000735],"VisaEBeam8":[0.0015,.00000707]
-                })
-                
+# dict for screens, their sizes, and resolution
```

Comment:
Be careful here - You're creating an OrderedDict using an unordered dict, so the order in your OrderedDict will still be unpredictable. * 

There are two ways to correct this. One is to instantiate OrderedDict with a list of (key, value) tuples. 
But much better is to avoid OrderedDicts altogether. If you're getting elements by integer location, better to be explicit by key. If you need an order during iteration, a list of (named)tuples or objects is generally better. 


\* in theory at least. In practice Python >=3.7 does help you out here.